Doc/library/datetime.rst

```diff
@@ -1678,7 +1678,7 @@ Usage of ``KabulTz`` from above::
 :class:`.time` Objects
 ----------------------
 
-A :class:`time` object represents a (local) time of day, independent of any particular
+A :class:`.time` object represents a (local) time of day, independent of any particular
```

Comment:
Nit: I don't think it's worth its own PR to make just this change, but FWIW, I would have just suppressed the link here, i.e.

```suggestion
A :class:`!time` object represents a (local) time of day, independent of any particular
```

The link is not useful, as it only takes the user two lines lower on the same page. It only adds visual clutter to the docs